sdks/wasm/Makefile

```diff
@@ -121,12 +122,15 @@ sample.dll: Simple.Dependency.dll sample.cs WebAssembly.Bindings.dll WebAssembly
 
 OPTIONS_CS = $(TOP)/mcs/class/Mono.Options/Mono.Options/Options.cs
 
-$(WASM_BCL_DIR)/monolinker.exe: $(TOP)/mcs/class/lib/wasm_tools/monolinker.exe
+$(WASM_TOOLS_DIR)/monolinker.exe: $(TOP)/mcs/class/lib/wasm_tools/monolinker.exe | $(WASM_TOOLS_DIR)
 	cp $< $@
 
-$(WASM_BCL_DIR)/Mono.Cecil.dll: Mono.Cecil.dll
+$(WASM_TOOLS_DIR)/Mono.Cecil.dll: Mono.Cecil.dll | $(WASM_TOOLS_DIR)
 	cp $< $@
 
+$(WASM_TOOLS_DIR):
```

Comment:
This is just **not** the way to go, you do _not_ create a folder in `sdks/out` from `sdks/wasm`. How do you make sure the archive contains it otherwise?

Since you **already** package `wasm_tools` profile in `$(TOP)/sdks/out/wasm-bcl/wasm_tools`, you can _way more simply_ define `WASM_TOOLS_DIR=$(TOP)/sdks/out/wasm-bcl/wasm_tools`. That's of course if you care to fit in the existing model.